CustomerManagementPanel/src/app/addresses/addresses-add/addresses-add.component.scss

```diff
@@ -0,0 +1,79 @@
+h1 {
+  text-align: center;
+  margin-bottom: 3rem;
+  font-size: 3rem;
+  font-weight: bold;
+}
+
+.addContainer {
+  text-align: center;
+  font-weight: bold;
+}
+
+button {
+  margin-left: 1rem;
+  margin-right: 1rem;
+  border: none;
+  font-size: 1.7rem;
+}
+
+.buttonC,
+.buttonS {
+  font-size: 1.9rem;
+  position: relative;
+  background: transparent;
+  border: none;
+  left: 0.5rem;
+  padding: 0.1rem 0.5rem;
+  transition: 0.3s transform;
+}
+
+.buttonC:hover {
+  transform: scale(1.2);
+  color: rgb(253, 77, 77);
```

Comment:
Probably you will use the same button colors in rest places on your site. Use colors.scss.